src/zkevm_specs/util/param.py

```diff
@@ -71,6 +71,7 @@
 
 
 COLD_SLOAD_COST = 2100
+COLD_ACCOUNT_ACCESS_COST = 100
```

Comment:
merge this part with the gas cost section in line 18-63, also add docs for each constant